src/main/java/com/capitalone/dashboard/auth/AuthProperties.java

```diff
@@ -185,8 +185,8 @@ public void applyDefaultsIfNeeded() {
 		}
 
 		if (getExpirationTime() == null) {
-			LOGGER.info("No JWT expiration time found in configuration, setting to one day.");
-			setExpirationTime((long) 1000 * 60 * 60 * 24);
+			LOGGER.info("No JWT expiration time found in configuration, setting to 30 minutes.");
+			setExpirationTime((long) 1000 * 60 * 30);
```

Comment:
can we make this configurable via properties?

Reply:
This is already configurable, by the auth.expirationTime in the properties file. The above code is for the default value if not set in the properties file.